index.html

```diff
@@ -28,6 +27,132 @@ <h1>Hemerson Foreste</h1>
           <button id="introduction-button" type="button">Start collaboration</button>
           <img src="images/image_geometry_2.svg" alt="image">
       </section>
+
+          <!--Projects Section-->
+          <section id="projects">
+            <h2>Projects</h2>
+            <div class="project-division">
+              <img src="images/Rectangle_21.png" alt="laptops">
+              <div class="project-name">
+                <h4>Project name goes here</h4>
+                <ul class="language-info">
+                    <li class="html-css">HTML/CSS</li>
+                    <li class="ruby">Ruby on Rails</li>
+                    <li class="javascript">JavaScript</li>
+                </ul>
+                <button type="button"><span>See this project</span><span><img class="arrow-right" src="images/Union.svg" alt="arrow right"></span></button>
+              </div>
+            </div>
+  
+            <div class="project-division">
+              <img src="images/MacBook.png" alt="laptops">
+              <div class="project-name">
+                <h4>Project name goes here</h4>
+                <ul class="language-info">
+                  <li class="html-css">HTML/CSS</li>
+                  <li class="ruby">Ruby on Rails</li>
+                  <li class="javascript">JavaScript</li>
+               </ul>
+               <button type="button"><span>See this project</span><span><img class="arrow-right" src="images/Union.svg" alt="arrow right"></span></button>
```

Comment:
- [ ] The button should and the button styling: Active, Hover, and Pressed. **_Check the other buttons on the page apply them to all of them._** [Requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/1_setup.md#:~:text=Implement%20the%20button%20interactions%20(enable%2C%20hover%2C%20etc.).)

Reply:
The reason I didn't use button styling: Active, Hover, and Pressed for the other buttons it's because I see only "Start collaboration" button on template.
![Buttons Primary Dark](https://user-images.githubusercontent.com/88809610/148473820-ca253f4f-e273-4dc3-8fae-53b8ac5d0405.png)
![Buttons Primary White](https://user-images.githubusercontent.com/88809610/148473824-c69d9533-0ab7-4913-a658-1bf91b99a6af.png)
![Buttons Secondary](https://user-images.githubusercontent.com/88809610/148473828-3fc7fd7d-92b0-49b2-b443-f6eee41e7b85.png)